src/components/progress-flow/progress-flow-item/partial-styles/_selected-indicator.scss

```diff
@@ -24,33 +23,30 @@
     .step {
         &.selected {
             &:before,
-            &:after {
+            .lock-icon {
                 content: '';
```

Comment:
```suggestion
```
it's better to move `content: '';` to line 33, as it belongs to the pseudo element, not the `lock-icon`.